hcl/eval/eval.go

```diff
@@ -118,37 +116,6 @@ func GetExpressionTokens(hcldoc []byte, filename string, expr hclsyntax.Expressi
 	return toWriteTokens(tokens), nil
 }
 
-// FromMapToObject converts a map of cty.Value to an object.
-// It is a programming error to pass any cty.Value that can't be
-// converted properly into an object field.
-func FromMapToObject(values map[string]cty.Value) cty.Value {
```

Comment:
Top :100: (I remember I copied this map conversion code from somewhere, was not aware of cty.ObjectVal)